sellotape/sellotape_dj/static/reconnecting-websocket.js

```diff
@@ -0,0 +1,382 @@
+// MIT License:
+//
+// Copyright (c) 2010-2012, Joe Walnes
```

Comment:
I am not familiar with that project, 
Why is it relevant to the chat application?
Maybe it should be on a separate PR for stating its purpose?

Reply:
This is a JS file that handles the messaging.
please read the documentation in it - it seems to be very clear.

Reply:
It is a JS project for web socketing, however, it doesn't state how it has relevance to our app, nor what it is supposed to do on chats.
To make it clearer, I was asking about it and suggesting to upload a followed PR with a relevant explanation on the subject.

Reply:
Again, the chat app can't run without reconnecting-websocket.js. 
You can read the documentation inside the file for more understanding.
besides if it still not clear please find attached the readme at this link -https://github.com/joewalnes/reconnecting-websocket.

Reply:
There is a difference between running and describing functionality, my question was asked to understand why that project was needed for the app. the documentation of this project described that the library is a web socket connection handler, however, could you elaborate on its use in the chat app?

Reply:
I already shared with you the files that explain your questions. 
let's take it offline if you have any more questions.